e2e/.env.example

```diff
@@ -0,0 +1,21 @@
+# Specify/override the NODE_ENV used for all services. Allowed values: `development`, `test`, `staging`, `production`
+NODE_ENV=development
+
+# Specify the URL-scheme to use for all services. Allowed values: `https` (=default) or `http` (for local development)
+SCHEME=http
+
+# A human-readable label for the environment. For example `Local DEV` or `Staging Pilot A`
+ENV_NAME=local
+
+# ------------
+# 121-service:
+# ------------
+# Specify a long/secure secret to enable the full reset of the database via an API-endpoint
+RESET_SECRET=fill_in_secret
+
+USERCONFIG_121_SERVICE_EMAIL_ADMIN=admin@example.org
+USERCONFIG_121_SERVICE_PASSWORD_ADMIN=password
```

Comment:
Would it be possible to load these values from the `services/.env`-file?

I think we can load/use `require('dotenv').config({ path: '../services/.env' })` somewhere?
Or load/source these specific values in the startup-script?